kuma/static/styles/includes/_vars.scss

```diff
@@ -103,45 +103,50 @@ $grid-width-column-main : -calc-col-width(9);
-
-$media-query-between-small-desktop-and-max-width : 'all and (min-width: #{$small-desktop-starts}) and (max-width: #{$max-width-default})';
-
-$media-query-large-desktop-and-up : 'all and (min-width: #{$small-desktop-ends})';
-$media-query-small-desktop-and-up : 'all and (min-width: #{$small-desktop-starts})';
-$media-query-tablet-and-up : 'all and (min-width: #{$tablet-starts})';
-$media-query-mobile-and-up : 'all and (min-width: #{$mobile-starts})';
-
-$media-query-small-desktop-only : 'all and (min-width: #{$small-desktop-starts})';
-$media-query-tablet-only : 'all and (min-width: #{$tablet-starts}) and (max-width: #{$tablet-ends})';
-$media-query-mobile-only : 'all and (min-width: #{$mobile-starts}) and (max-width: #{$mobile-ends})';
-$media-query-small-mobile-only : $media-query-small-mobile;
-
-$navigation-break-ends : 1100px;
-$navigation-block-ends : 850px;
-
-$navigation-break-starts : $navigation-block-ends + 1px;
-
-$media-query-navigation-break : 'all and (max-width: #{$navigation-break-ends})'; /* Moves nav items all the way to the left, moves logo above */
-$media-query-navigation-block : 'all and (max-width: #{$navigation-block-ends})'; /* Moves nav menu items to block, its own line */
-
-$media-query-break-and-up : 'all and (min-width: #{$navigation-break-starts})';
+/* break points */
+$max-width-default : (1400px / 16px) * 1em;
+$small-desktop-ends : (1200px / 16px) * 1em;
+$tablet-ends : (1024px / 16px) * 1em;
+$mobile-ends : (768px / 16px) * 1em;
+$small-mobile-ends : (480px / 16px) * 1em;
+
+$small-desktop-starts : $tablet-ends + .001em;
+$tablet-starts : $mobile-ends + .001em;
+$mobile-starts : $small-mobile-ends + .001em;
+
+$navigation-break-ends : (1100px / 16px) * 1em;
+$navigation-block-ends : (850px / 16px) * 1em;
+
+$navigation-break-starts : $navigation-block-ends + .001em;
+
+/* media queries */
+$mq-max-width-and-up : 'all and (min-width: #{$max-width-default})';
+$mq-large-desktop-and-up : 'all and (min-width: #{$small-desktop-ends})';
+$mq-small-desktop-and-up : 'all and (min-width: #{$small-desktop-starts})';
+$mq-tablet-and-up : 'all and (min-width: #{$tablet-starts})';
+$mq-mobile-and-up : 'all and (min-width: #{$mobile-starts})';
+
+$mq-small-desktop-only : 'all and (min-width: #{$small-desktop-starts})';
+$mq-tablet-only : 'all and (min-width: #{$tablet-starts}) and (max-width: #{$tablet-ends})';
+$mq-mobile-only : 'all and (min-width: #{$mobile-starts}) and (max-width: #{$mobile-ends})';
+$mq-small-mobile-only : 'all and (max-width: #{$small-mobile-ends})';
+
+$mq-navigation-break : 'all and (max-width: #{$navigation-break-ends})'; /* Moves nav items all the way to the left, moves logo above */
+$mq-navigation-block : 'all and (max-width: #{$navigation-block-ends})'; /* Moves nav menu items to block, its own line */
+
+$mq-break-and-up : 'all and (min-width: #{$navigation-break-starts})';
+
+/* avoid using these media queries, code mobile first */
```

Comment:
Yeah, a lot of them will go with the header and footer changes in the next few weeks. And more will go in July and Aug with the content restructuring. After that it might be possible to convert the remaining examples.